crates/orchestrator/src/lib.rs

```diff
@@ -340,15 +355,63 @@ where
     // Assumes nodes do not post 'ready' twice
     // TODO ED Add a map to verify which nodes have posted they're ready
     fn post_ready(&mut self) -> Result<(), ServerError> {
+        if !self.accepting_connections {
```

Comment:
I don't think we want this here.  We should stop allowing people to register new keys once we've restarted.  We still want people to post whether or not they're ready, though.  I think we want to put this in the register keys function.